app/components/Card/CardBack.tsx

```diff
@@ -24,21 +25,23 @@ const CardBack: React.FC<CardBackProps> = ({
   setIsFlipped,
   name,
   description,
+  path,
 }) => {
+  const style =
+    path === "/" ? responsiveValues.carousel : responsiveValues.machinesPage;
+
   const handleClick = () => {
     setIsFlipped(!isFlipped);
```

Comment:
update this to: `setIsFlipped((prevState) => !prevState);` to ensure that latest state is used (e.g. if user clicks rapidly)